README.md

```diff
@@ -1,18 +1,27 @@
 # Bitstyles
 
-A developing collection of CSS styles and helpers, for use in Bitcrowd projects.
+A collection of tiny Sass CSS objects and helpers, Bitstyles is intended to be used by reasonably CSS- & HTML-aware frontend developers. This is no simple drop-in solution to skin a website, like [Bootstrap](http://getbootstrap.com) or [Foundation](http://foundation.zurb.com), rather a collection of focused object-classes, each defining just a single visual aspect. By using these simple building blocks in your HTML, it’s possible to construct complex layouts without repeating CSS.
 
-## Using Bitstyles in a project
+For a certain set of elements in your UI, this strategy of applying multiple atomic classes eventually becomes untenable. This is generally for one of two reasons:
+- the elements are very often-repeated, so using one class instead of many would reduce chances of inconsistency or errors e.g. buttons, cards
+- the number of object classes on an element becomes unreadable in the HTML. This often also results in the appearance of the element becoming hard to reason about just from the HTML source, negating a large advantage of this approach in the first place.
+
+For components which match these criteria it may make more sense to create component classes in your project to compose several of the simpler object classes (and whatever styling you may need on top). For this reason, all the object classes are also provided as Sass mixins — `@include` them in your own Sass components as needed.
 
-Bitstyles can be added to your project via [npm](https://www.npmjs.com/) or [Bower](http://bower.io/), as each project demands.
+Again unlike Bootstrap and Foundation, there are no javascript widgets or plugins included in Bitstyles.
 
-The source repository is currently private, and is not registered with the Bower
-or npm registries. You will therefore need to install directly from a tagged
-Github release (not from master), and you will need read access to the repo.
+See the [Bitcrowd frontend styleguide](https://github.com/bitcrowd/frontend) for more details of the technical principles on which Bitstyles is built. The TL;DR: it’s a variation of the BEMATOMICOOCSS methodology:
```

Comment:
:D Yup! It’s tongue-in-cheek but there’s no other name for this mix of conventions yet